src/main/scala/firrtl/passes/RemoveCHIRRTL.scala

```diff
@@ -56,7 +56,7 @@ object RemoveCHIRRTL extends Pass {
       (e1s zip e2s) map { case (e1, e2) => Mux(e.cond, e1, e2, mux_type(e1, e2)) }
     case (e: ValidIf) =>
       create_exps(e.value) map (e1 => ValidIf(e.cond, e1, e1.tpe))
-    case (e) => (e.tpe) match {
+    case (e) => e.tpe match {
```

Comment:
The case statements are going to be handle by a separate cleanup pass. I haven't touched any of that stuff yet.  I am not sure why these got wrapped in parens int the first place.